### ISSUE TITLE
Use `set_transaction_id` method of WC_Order to set transaction ID.

### ISSUE THREAD

**Issue by iamdharmesh** [godaddy-wordpress/wc-plugin-framework#570]
This is an enhancement request. Framework uses `update_post_meta` to set transaction id to order, and it can create an issue for the upcoming "Custom Order Table" functionality of WooCommerce.
https://github.com/godaddy-wordpress/wc-plugin-framework/blob/master/woocommerce/payment-gateway/class-sv-wc-payment-gateway.php#L2517

[set_transaction_id](https://woocommerce.github.io/code-reference/classes/WC-Order.html#method_set_transaction_id) method should be used here as a more future-proof solution. 

Example:
```php
$order->set_transaction_id( $response->get_transaction_id() );
$order->save();
```

Thanks

**Comment by unfulvio-godaddy** [comment]
Hey @iamdharmesh sorry for the late reply. We have plans to add support to HPOS soon in the FW and plugins adopting it. All functions referencing WordPress own `*_post_meta` will be updated accordingly.

**Comment by unfulvio-godaddy** [comment]
fixed by #584
